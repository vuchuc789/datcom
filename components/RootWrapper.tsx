import { EmotionRegistry } from '../lib/EmotionRegistry';
import { ThemeProvider } from '../contexts/ThemeContext';

type RootWrapperProps = React.PropsWithChildren;

export const RootWrapper: React.FC<RootWrapperProps> = ({ children }) => {
  return (
    <EmotionRegistry>
      <ThemeProvider>{children}</ThemeProvider>
    </EmotionRegistry>
  );
};
