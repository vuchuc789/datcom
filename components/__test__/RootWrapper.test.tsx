import { render } from '@testing-library/react';
import { RootWrapper } from '../RootWrapper';

describe('RootWrapper', () => {
  test('render correctly', () => {
    const { container } = render(<RootWrapper>sample content</RootWrapper>);

    expect(container).toMatchSnapshot();
  });
});

export {};
