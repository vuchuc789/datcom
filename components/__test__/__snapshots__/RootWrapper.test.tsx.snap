// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RootWrapper render correctly 1`] = `
<div>
  sample content
</div>
`;
